pkg/generic/generic.go

```diff
@@ -0,0 +1,94 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package generic provides functions and steps implementations not specifically related to Kubernetes nor AWS.
+package generic
```

Comment:
nit: should we just name this common?

Reply:
Could be, I thought about using that but did not do it thinking that `common` feels like something other pkgs will use or have in common and this is a standalone pkg that not necessarily will be used by the other pkgs. I can go with common if you feel strongly about it. Let me know